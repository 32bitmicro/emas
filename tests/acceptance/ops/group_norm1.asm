LW r0, r0
TW r0, r0
LS r0, r0
RI r0, r0
RW r0, r0
PW r0, r0
RJ r0, r0
IS r0, r0
BB r0, r0
BM r0, r0
BS r0, r0
BC r0, r0
BN r0, r0
OU r0, r0
IN r0, r0
