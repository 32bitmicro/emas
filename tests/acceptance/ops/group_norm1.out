@ 0x0000 : 0x4000  /  010 000 0 000 000 000  /  16384
@ 0x0001 : 0x4400  /  010 001 0 000 000 000  /  17408
@ 0x0002 : 0x4800  /  010 010 0 000 000 000  /  18432
@ 0x0003 : 0x4c00  /  010 011 0 000 000 000  /  19456
@ 0x0004 : 0x5000  /  010 100 0 000 000 000  /  20480
@ 0x0005 : 0x5400  /  010 101 0 000 000 000  /  21504
@ 0x0006 : 0x5800  /  010 110 0 000 000 000  /  22528
@ 0x0007 : 0x5c00  /  010 111 0 000 000 000  /  23552
@ 0x0008 : 0x6000  /  011 000 0 000 000 000  /  24576
@ 0x0009 : 0x6400  /  011 001 0 000 000 000  /  25600
@ 0x000a : 0x6800  /  011 010 0 000 000 000  /  26624
@ 0x000b : 0x6c00  /  011 011 0 000 000 000  /  27648
@ 0x000c : 0x7000  /  011 100 0 000 000 000  /  28672
@ 0x000d : 0x7400  /  011 101 0 000 000 000  /  29696
@ 0x000e : 0x7800  /  011 110 0 000 000 000  /  30720
